core/scss/components/main-nav/_main-nav--light.scss

```diff
@@ -0,0 +1,165 @@
+@charset "UTF-8";
+
+.su-main-nav.su-main-nav--light {
+
+  .su-main-nav__toggle {
+    &[aria-expanded="true"] {
+      &::before {
+        background-color: $color-driftwood;
+      }
+
+      &:hover {
+        &::before {
+          background-color: $color-dark-red;
+        }
+      }
+    }
+  }
+
+  ul {
+    background-color: $color-white;
+  }
+
+  > ul {
+
+    @include grid-media('lg') {
+      background-color: transparent;
+
+      > li {
+        > a {
+          &:hover,
+          &:focus {
+            color: $color-dark-red;
+
+            &::before {
+              background-color: $color-dark-red;
+            }
+          }
+
+          &:active {
+            color: $color-black;
+
+            &::before {
+              background-color: $color-black;
+            }
+          }
+
+          &[aria-expanded="true"] {
+            color: $color-black;
+
+            &::before {
+              background-color: $color-driftwood;
+            }
+
+            &:hover {
+              color: $color-dark-red;
+
+              &::before {
+                background-color: $color-dark-red;
+              }
+            }
+          }
+        }
+
+        &.su-main-nav__item--current {
+          > a {
+            color: $color-black;
+
+            &::before {
+              background-color: $color-driftwood;
+            }
+
+            &:hover,
+            &:focus {
+              color: $color-dark-red;
+
+              &::before {
+                background-color: $color-dark-red;
+              }
+            }
+
+          }
+        }
+      }
+    }
+  }
+
+  li {
+    a {
+      color: $color-black;
+      border-bottom-color: #d9d9d9;
+
+      &:hover,
+      &:focus {
+        &::before {
+          background-color: $color-black;
+        }
+      }
+
+      &:active {
+        color: $color-dark-red;
+
+        &::before {
+          background-color: $color-dark-red;
+        }
+      }
+
+      &[aria-expanded="true"] {
+        &::before {
+          background-color: $color-dark-red;
+        }
+      }
+    }
+
+    &.su-main-nav__item--current {
+      > a {
+        color: $color-dark-red;
+
+        &::before {
+          background-color: $color-dark-red;
+        }
+      }
+    }
+
+    &.su-main-nav__item--parent {
+      > a {
+        &::after {
+          background: url("#{$image-path}/plus-black.svg") no-repeat 0 0;
+          background-size: 100%;
+          content: "";
```

Comment:
This seems to be the same as the default style. Do we need this here?

Reply:
https://github.com/SU-SWS/decanter/pull/264/files#diff-9e1da243c95cc0238217fd5a27be523fR142

Reply:
I cleaned up some of the other properties, but the image file is different (black vs. white).